design/schedule-unpaused-triggers-config_design.md

```diff
@@ -0,0 +1,108 @@
+# Schedule Unpaused Triggers Config Design
+## Abstract
+User want to configure what happens when schedule is unpaused. Either run backup immediately or at the next cron schedule.  This design proposes to add a new field to the schedule spec to allow user to configure the behavior
```

Comment:
Do we really want to make it a configuration item? IMO, only taking backup at the next cron schedule is more reasonable. If users want to run backup immediately, they can trigger it by creating a backup from the schedule manually.
IMO, I'd prefer to avoid exposing more and more configuration items to the end users

Reply:
So I probably need to reword
> Either run backup immediately

The ask from the user issue was that backup was unintentionally running immediately after unpausing caused by the current logic being based on LastBackupTimestamp.

Schedule that has been backedup with a timestamped then paused, and long enough time has passed, unpausing can make backup go off immediately.

User wants ability to make it not go off when unpausing but wait till next cron after unpause timestamp.

@rnarenpujari am I summarizing the need here pretty well?

Reply:
I understand the requirement and I think the design of this PR has already resolved the issue properly.
What I want to discuss is whether it is necessary to make taking backup immediately or not to be configurable.  Could we just make the change that doesn't take backup immediately with the solution in this design?